packages/syft/src/syft/node/node.py

```diff
@@ -1458,14 +1463,35 @@ def add_queueitem_to_queue(
             return result
         return job
 
+    def _sort_jobs(self, jobs: list[Job]) -> list[Job]:
+        job_datetimes = {}
+        for job in jobs:
+            try:
+                d = datetime.strptime(job.creation_time, DATETIME_FORMAT)
+            except Exception:
+                d = datetime(MINYEAR, 1, 1)
+            job_datetimes[job.id] = d
+
+        jobs.sort(
```

Comment:
if we have a running job and a failed job, maybe we should show the running one?